src/_guides/openapi/specification/v3.1/understanding-structure/http-requests.md

```diff
@@ -0,0 +1,105 @@
+- Descriptions for these requests to add context to API documentation.
+
+Let's consider the [Train Travel API](https://bump.sh/blog/modern-openapi-petstore-replacement), which allows users to book train tickets. 
+
+### Creating a Resource
+
+When a user wants to book a train ticket, they need to send details like the passenger's name, trip ID, date of travel, and seat preference, which would look a bit like this:
+
+```yaml
+paths:
+  /bookings:
+    post:
+      summary: Book a train ticket
+      description: Endpoint to book a train ticket
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                passenger_name:
+                  type: string
+                  example: "John Doe"
+                trip_id:
+                  type: string
+                  example: "1234"
+                date:
+                  type: string
+                  format: date
+                  example: "2024-08-15"
+                seat_preference:
+                  type: string
+                  enum: [window, aisle, any]
+                  example: "window"
+```
+
+Here the `requestBody` object defines two important properties:
+
+- `required: true` - indicates that the request body is mandatory for this operation.
+  
+- `content` - specifies that the request body should be in `application/json` format with the following `schema`.
+
+The schema defines the structure of the request body, including properties like passenger_name, train_id, `date`, and `seat_preference`. This can be defined inline like this example, or it can use `components` to share an [existing schema](../data-models/schema-and-data-types.md) and reduce repetition. 
+
+### Updating a Resource
+
+If a user wants to update their booking (e.g.: change the seat preference), the API can define a PUT or PATCH response, to allow updating the entire booking, or part of the booking. Either way, they need to send the updated data in the request body. Here’s how to define it:
+
+```yaml
+paths:
+  /bookings/{bookingId}:
+    patch:
+      summary: Update a booking
+      description: Endpoint to update an existing booking
+      parameters:
+        - name: bookingId
+          in: path
+          required: true
+          schema:
+            type: string
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                seat_preference:
+                  type: string
+                  enum: [window, aisle, any]
+                  example: "aisle"
+```
+
+Here the `PATCH` method is used to update the one specific field that needs updating in the existing booking. The `required: true` says the `requestBody` is mandatory, and the only media type defined is `application/json` so that says the request must be in that format.
```

Comment:
```suggestion
Here the `PATCH` method is used to describe an operation that can update one specific field from an existing booking. The `required: true` says the `requestBody` is mandatory, and the only media type defined is `application/json` so that says the request must be in that format.
```